### DISCUSSION TITLE
Friend needs help

### DISCUSSION THREAD

**Discussion by JackPrikker** [community/community#79141] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi there, sometime around late October my friends account account got suspended without warning and he was hit with a "Access to your account has been suspended due to a violation of our TOS" They think it might have been because they was messing with codespaces as someone else seems to have also had the same problem before where too many http request got them kicked, when researching the issuse on the community post earlier and if so they apolize as they didn't mean any harmful intentions.

Of course after than they tried to make a ticket and try to resolve the issue as they had no actually harmful intentions but that ticket was send on October 27 and they have yet to get a response back.

their original account is github.com/hinx2 and ticket id is: 2402164

so now I am here am wondering if anyone is able to help or escalate the issue? or is there anything else that they could do help with the support process themself?

thanks for the any help in advance

**Comment by queenofcorgis** [comment]
Hi there @JackPrikker, 

I checked in on this for you - though be advised the community is unable to assist or escalate in these matters. An important part of the restoration process is for security purposes, we can discuss account details only when we have established that we are in contact with the account's legitimate owner. To do this, we must receive communication from a current, verified email address associated with the account in question.
